### PR TITLE
Create ﻿Ambiguous-Squids-Sprint-2-Commitment-February-8-2017.md

### PR DESCRIPTION
**Pull request by mccordgh**
## Description
Create the team sprint commitment for the week of Feb 7 to Feb 14, 2017.

### CODE REVIEW

**Review by asimonia — APPROVED** [review]
🐤

File: ﻿Ambiguous-Squids-Sprint-2-Commitment-February-8-2017.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Ambiguous Squids Sprint Commitment February 8, 2017
+  
+## Client
+Steve @ BrownBiguity Inc.
+
+## Squids
+Nick Chemsak, Alex Simonian, Matthew McCord, Dani Adkins, Richard Whitfield
```
Comment (rtwhitfield84) [inline ﻿Ambiguous-Squids-Sprint-2-Commitment-February-8-2017.md]: we miss dani, but now we have Ali!

**Review by nchemsak — APPROVED** [review]
👍

**Review by rtwhitfield84 — CHANGES_REQUESTED** [review]
Add Ali to the squids lineup and remove dani, otherwise looks good

**Review by rtwhitfield84 — COMMENTED** [review]
⚡️ 👍

**Comment by ghost** [comment]
👍

**Review by rtwhitfield84 — APPROVED** [review]

**Pull request merged**